src/actions/cubeActions.js

```diff
@@ -46,7 +46,7 @@ export function countryIsSelected(idxOfSelector, selectedCountry, graphId) {
 export function fetchData(country, risk, graphId, test=false) {
   return function(dispatch) {
     dispatch(requestData(country, risk, graphId))
-    let ENDPOINT = `/api/v1/count_by_country?limit=500&country=${country}&risk=${risk}`
+    let ENDPOINT = `/api/v1/count_by_country?limit=500&country=${country}&risk=${risk}&drilldown=as`
```

Comment:
are we using same action for getting data for barcharts on the right, that we use for getting data for line graphs on right? 
If so what about normalized counts? this API call does not include them, yet.